pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -28,6 +30,7 @@ const (
 	libvirtDriverVersion        = "0.12.5"
 	crcDnsmasqConfigFile        = "crc.conf"
 	crcNetworkManagerConfigFile = "crc-nm-dnsmasq.conf"
+	minSupportedLibvirtVersion  = "3.4.0"
```

Comment:
Add a comment that this should be kept in sync with code-ready/machine-driver-libvirt go.mod?

Reply:
@cfergeau done